README.md

```diff
@@ -147,6 +147,13 @@ Returns a thunk which when executed deletes the entire database.
 yield db.drop();
 ```
 
+#### list
+Returns a thunk which when executed list existing collections in the database.
+
+```js
+yield db.list();
```

Comment:
to me, "list" is a bit too vague as there are also indexes. how about "collections" or "collectionNames"?